.github/SECURITY.md

```diff
@@ -8,6 +8,8 @@ Sangoma takes the security of our software products and services seriously. We a
 
 If you believe you have found a security vulnerability please report it to us through the Security reporting process on GitHub.
 
+For FreePBX, use the "Report a vulnerability" button at the top of the [FreePBX Security Reporting repository](https://github.com/FreePBX/security-reporting/security).
```

Comment:
As communicated on the issue, I am trying not to fork the policy across things and use a single unified policy across everything. I will consider this change and whether I'm comfortable forking the policy, even in this smallest amount.